### PR TITLE
Barnabam/dataflow

### PR DESCRIPTION
**Pull request by Vaticinator**
VTK demo models (Unicorn <3)

### CODE REVIEW

**Comment by Vaticinator** [comment]
You can overvrite my conflict files:
unity/Holo/Assets/GFX/Materials/DefaultModel.mat
unity/Holo/Assets/Scenes/AnimatedModel.unity

File: unity/Holo/Assets/GFX/Models/line.prefab [hunk 1]
```diff
@@ -0,0 +1,128 @@
+%YAML 1.1
```
Comment (ellendil) [inline unity/Holo/Assets/GFX/Models/line.prefab:1]: Do we need this file?

File: unity/Holo/Assets/GFX/Materials/ShaderTest.mat [hunk 2]
```diff
@@ -0,0 +1,79 @@
+%YAML 1.1
```
Comment (ellendil) [inline unity/Holo/Assets/GFX/Materials/ShaderTest.mat:1]: Do we need this file?

**Review by ellendil — APPROVED** [review]
LGTM, please check if the files are needed.

**Pull request merged**